src/layouts/govuk.njk

```diff
@@ -5,7 +5,6 @@
 {% extends "govuk-frontend/template.njk" %}
 
 {% set htmlLang = "en" %}
-{% set themeColor = "blue" %}
```

Comment:
`black` might be good, as that's the colour of the GOV.UK header

Reply:
It's actually `#0b0c0c`, but yeah, let's go with that.

Reply:
Conversation with Luke:

```
Richard Towers  [24 minutes ago]
You mean sets the colour to govuk-black?


Richard Towers  [24 minutes ago]
I’m not sure whether we should remove the theme colour entirely or set it black like GOV.UK does :thinking2:

Richard Towers  [23 minutes ago]
`blue` is definitely out though :blue-cap-crew:

luke.malcher  [19 minutes ago]
default is the best imo. ...
```

Suggests just removing it, so I've switched back to that.

Reply:
(Although `#0b0c0c` is the default in govuk-frontend, so it will still be black)